src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Figure {
```

Comment:
The name 'Figure' for an interface is a noun, which is good practice, but the interface itself should not contain methods that are conceptually different from each other like 'getArea()' and 'draw()'. Consider splitting this interface into two separate interfaces, each with a single responsibility.